frappe/tests/ui_test_helpers.py

```diff
@@ -333,3 +349,86 @@ def insert_child(doc, data, barcode, check, rating, duration, date):
 	insert_child(doc, "Drag", "08189DIHAA2981", 0, 0.7, 342628, "2022-05-04")
 
 	doc.insert()
+
+
+@frappe.whitelist()
+def setup_tree_doctype():
+	frappe.delete_doc_if_exists("DocType", "Custom Tree")
```

Comment:
Do we need to check for permission or `frappe.flags.in_test` in these whitelisted methods??

Reply:
I don't think so it would be necessary since I am not changing any existing doctype, just adding a few for testing in cypress